### ISSUE TITLE
STUDENT T-TEST PAIRED/ Discrepancies in calculation with MS Excel / Changes not visible in 2.2.3

### ISSUE THREAD

**Issue by tp-development** [GBonnaire/jspreadsheet-plugins-and-editors#32]
### Plugin Name with version

2.2.3

### JSpreadsheet version

v9.x

### Describe the bug

After attaching 2.2.3 the changes are not visible (lack of the new parameter).
Checked after cache clearing and in on incognito window.

### Reproduction steps

```bash
1.
2.
3.
...
```

**Comment by GBonnaire** [comment]
I have check and mn my version 2.2.3, i have this :

**Comment by GBonnaire** [comment]
<img width="765" alt="image" src="https://github.com/GBonnaire/jspreadsheet-plugins-and-editors/assets/52194475/e8702d66-73ce-466a-8028-38dc171354fe">

**Comment by GBonnaire** [comment]
Do you have a custom documentation ? check this 😊

**Comment by tp-development** [comment]
Yes, I do have a custom docs, but I don't see it in the parameters in the main line) either.

**Comment by tp-development** [comment]
![image](https://github.com/GBonnaire/jspreadsheet-plugins-and-editors/assets/98662312/69cff102-6e24-4d2f-9adc-0150c82401a0)

**Comment by GBonnaire** [comment]
Hi,

if you use a custom documentation, please look your custom documentation and make change. If not change this documentation about this function, please try to reload cache

**Comment by tp-development** [comment]
I have tried to reload the cache and it didn't work. The issue is not in the custom documentation part but in the parameter list. 
We plan the release on the weekend, this is the last issue, we are waiting for. If we got the fix, we could complete and release the feature.

**Comment by GBonnaire** [comment]
I have check my code, and this parameter is not in documentation.

Can you share your custom documentation (only TTEST.TWOSAMPLES.T ?

If you have access to jspreadsheet var in console, show : `jspreadsheet.formulasDoc['TTEST.TWOSAMPLES.T()']`

You have 6 properties : 
- syntax (the first element in gray), it's here there is a list of parameters where you and me, we see a different things
- about
- parameters
- example
- resultexample
- link
Can you check that ?

```javascript
jspreadsheet.formulasDoc['TTEST.TWOSAMPLES.T()'] = {
                        syntax: "TTEST.TWOSAMPLES.T(firstColumn &rarr; Range, secondColumn &rarr; Range, [pairedSamples = FALSE])",
                        about: "Return tStatistic value of studentsTTestTwoSamples<br>Statistical tests - parametric: Computes Student’s two sample t-test for the values of two variables given by firstColumn and secondColumn.",
                        example: "=TTEST.TWOSAMPLES.T([204, 212, 199, 209, 191, 190, 223, 185, 206, 212],[189, 199, 191, 182, 176, 180, 220, 182, 194, 216])",
                        resultexample: "1.69068…"
                };
```

On version minised (published)
<img width="1258" alt="image" src="https://github.com/GBonnaire/jspreadsheet-plugins-and-editors/assets/52194475/fb6be3b6-4fab-4238-9a63-fe0a4276cb3f">

**Comment by tp-development** [comment]
Ok, after checking with the developer, it seems that we have these changes. However, the dependant/pairedSample parameter does not work - whatever value we pass TRUE/FALSE/true/false/1/0 it calculates as if it were two independent samples. Let me know if I need to create another issue, or we just proceed with current one.

**Comment by GBonnaire** [comment]
i have checked and tried to reproduct error. I don't success.

Can you use option this option on jspreadsheet : 

`debugFormulas: true,`

and share console error ?

**Comment by GBonnaire** [comment]
Can you share your option on load jspreadsheet ?

**Comment by tp-development** [comment]
![image](https://github.com/GBonnaire/jspreadsheet-plugins-and-editors/assets/98662312/2efd81fd-5b7b-42a0-97ae-7cbbffd209ae)
![image](https://github.com/GBonnaire/jspreadsheet-plugins-and-editors/assets/98662312/0e10fd24-5a2b-4024-a568-51f159ff16f8)

These are the console logs for 
=TTEST.TWOSAMPLES.T(A1:A100,B1:B100,TRUE)
=TTEST.TWOSAMPLES.DF(A1:A100,B1:B100,TRUE)
let me know if this helps.

In columns A1:A100, B1:B100 we have solely integer values.

**Comment by GBonnaire** [comment]
yes, i have received this information with PK. Can you share your option of JSS ?

because, i don't reproduct your error :/ 

I have check with same version of JSS, same version of formula and same versions of plugins. No error

**Comment by GBonnaire** [comment]
Fixed on 2.2.7
